Packs/MISP/Integrations/MISPV3/MISPV3_description.md

```diff
@@ -0,0 +1,20 @@
+## MISP V3
+
+## Get Tag IDs
+In MISP V3, indicator scoring calculated depending on **MISP's tags**.
```

Comment:
dont forget to add that score will use threat level ids to score if tags are not configured and will regard 1,2,3 values of threat level ids as malicious.